templates/user-data.sh.tmpl

```diff
@@ -41,7 +41,7 @@ sudo docker run \
 %{ endif ~}
   -v $${CONFIG_DIR}:/snowplow/config \
   -p ${port}:${port} \
-  -e 'JAVA_OPTS=-Dorg.slf4j.simpleLogger.defaultLogLevel=info -Dcom.amazonaws.sdk.disableCbor' \
+  -e 'JAVA_OPTS=${java_opts}' \
```

Comment:
`-Dcom.amazonaws.sdk.disableCbor` [looks pretty important here](https://github.com/aws/aws-sdk-java/issues/2493), maybe we shouldn't risk losing it? If users want to change the loglevel they could just set `-Dorg.slf4j.simpleLogger.defaultLogLevel=somethingelse` and it should override the first option. @archeus, thoughts?

```suggestion
  -e 'JAVA_OPTS=-Dorg.slf4j.simpleLogger.defaultLogLevel=info -Dcom.amazonaws.sdk.disableCbor ${java_opts}' \
```

Reply:
yes @andrein, I agree, we should keep `-Dcom.amazonaws.sdk.disableCbor` here so we don't accidentaly lose it when adding other java opts